perf/src/main/java/org/bf2/performance/framework/ManagedKafkaStateAssertionParameterResolver.java

```diff
@@ -69,7 +69,7 @@ public static Optional<Integer> getLargestKnownInstanceCount(TestInfo info) {
     }
 
     private static Stream<ManagedKafkaAssertion> getManagedKafkaAssertionStream(OpenshiftClusterData openshiftClusterData, Optional<Method> testMethod) {
-        return Arrays.stream(testMethod.get().getAnnotationsByType(ManagedKafkaAssertion.class))
+        return Arrays.stream(testMethod.orElse(null).getAnnotationsByType(ManagedKafkaAssertion.class))
```

Comment:
Here you'll want to conditionally map the method to a stream of its `ManagedKafkaAssertion` annotations. Something like the following. Instead of `null`, the default will be an empty stream.
```java
return testMethod
    .map(m -> m.getAnnotationsByType(ManagedKafkaAssertion.class))
    .map(Arrays::stream)
    .orElseGet(Stream::empty)
    .filter(a -> a.instanceType().equals(openshiftClusterData.getWorkerInstanceType()))
    .sorted(Comparator.comparingInt(ManagedKafkaAssertion::minWorkers))
    .collect(Collectors.toList())
    .stream();
```